modules/default/newsfeed/newsfeed.njk

```diff
@@ -13,13 +13,15 @@
         <div class="newsfeed-title bright medium light{{ ' no-wrap' if not config.wrapTitle }}">
             {{ title }}
         </div>
-        <div class="newsfeed-desc small light{{ ' no-wrap' if not config.wrapDescription }}">
-            {% if config.truncDescription %}
-                {{ description | truncate(config.lengthDescription) }}
-            {% else %}
-                {{ description }}
-            {% endif %}
-        </div>
+        {% if (config.showDescription) %}
```

Comment:
I am wondering whats occuring more often "if (<condition>)" or "if <condition>" ?

Reply:
I'd say we strip out all unnecessary chars. But that isn't specifically necessary for this PR.